lib/index.js

```diff
@@ -11,6 +11,7 @@ var crypto = require('crypto');
 var database = require('bedrock-mongodb');
 var util = require('util');
 var BedrockError = bedrock.util.BedrockError;
+require('bedrock-server');
```

Comment:
@davidlehn -- can we solve this problem with some kind of new dynamic config thing?